docs/FrameDescriptions.md

```diff
@@ -1,21 +1,121 @@
-Following are the descriptions of the Frames used by different devices in our framework following these issues:
+**Following you can find the descriptions of the frames used by different devices in teleoperation framework:**
 
-**Oculus Device**
+## Inertial Frame
+
+We define the inertial frame in the human side as following:
```

Comment:
Maybe we can specify that this frame is fixed and we assume it to be in a specific position and orientation (e.g. corresponding to the person head at startup, oriented with the x pointing forward and the z pointing upward, or something more formal)